packages/infolists/resources/lang/en/components.php

```diff
@@ -3,7 +3,8 @@
 return [
 
     'text_entry' => [
-        'more_list_items' => 'and :count more',
+        'less_list_items' => 'Show :count less',
+        'more_list_items' => 'Show :count more',
```

Comment:
I think we should/could add this functionality to checkbox lists as well.
I'm mentioning this now since that'd make it easier to extract the translations instead of doing that later.

Reply:
Checkbox lists don't have limits at the moment, and I don't think this needs transferring yet